File: Beginner/1010 (Simple Calculate)/1010.py
a,b,e = input().split(" ")
a = int(a)
b = int(b)
e = float(e)

c,d,f = input().split(" ")
c = int(c)
d = int(d)
f = float(f)

print ("VALOR A PAGAR: R$ ""{:.2f}".format(e*b + d*f))

File: Beginner/1010 (Simple Calculate)/1010.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] texto = Console.ReadLine().Split(' ');
            int a = int.Parse(texto[0]), b = int.Parse(texto[1]);
            double e = double.Parse(texto[2]);
            string[] texto2 = Console.ReadLine().Split(' ');
            int c = int.Parse(texto2[0]), d = int.Parse(texto2[1]);
            double f = double.Parse(texto2[2]);
            double result = e * b + d * f;
            Console.WriteLine("VALOR A PAGAR: R$ " + string.Format("{0:0.00}", result));



        }
    }
}